.github/workflows/publish.yml

```
@@ -67,11 +67,21 @@ jobs:
     name: Docker Hub
     runs-on: ubuntu-24.04
     if: ${{ needs.check.outputs.released == 'false' }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: docker
+            registry: docker.io
```

Looks like [`docker/login-action`](https://github.com/docker/login-action) uses an missing/value for logging in to Docker Hub, so I think we should use that instead of guessing the registry value.

```suggestion
            registry: ""
```